src/IntegratedExperts/Behat/Relativity/RelativityExtension.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * @file
+ * Behat relativity extension.
+ */
+
+namespace IntegratedExperts\Behat\Relativity;
+
+use Behat\Behat\Context\ServiceContainer\ContextExtension;
+use Behat\Testwork\ServiceContainer\Extension as ExtensionInterface;
+use Behat\Testwork\ServiceContainer\ExtensionManager;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use WebDriver\Exception;
+
+/**
+ * Class RelativityExtension.
+ */
+class RelativityExtension implements ExtensionInterface
+{
+
+    /**
+     * Extension configuration ID.
+     */
+    const MOD_ID = 'integratedexperts_relativity';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigKey()
+    {
+        return self::MOD_ID;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function initialize(ExtensionManager $extensionManager)
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(ArrayNodeDefinition $builder)
+    {
+        $builder->
+            children()
+                ->arrayNode('components')->
+                    useAttributeAsKey('key')->
+                        info('Name of component: shouldn\'t content dash "-" or remember it transformed to underscore "_" ')->
+                        isRequired()->
+                    prototype('variable')->
+                      isRequired()->
+                    end()->
+                end()
+                ->scalarNode('offset')->
+                    defaultValue(0)->
+                end()
+                ->arrayNode('breakpoints')->
+                    useAttributeAsKey('key')->
+                        info('Name of devise or screen size.')->
+                        isRequired()->
+                    arrayPrototype()->
+                        children()->
+                            scalarNode('width')->
+                                info('Screen width.')->
+                                isRequired()->
+                                    end()->
+                            scalarNode('height')->
+                                isRequired()->
+                                    info('Screen height.')->
+                                    end()->
+                            booleanNode('default')->
```

Comment:
this actually requires custom validator to be implemented: this value MUST be set for one and one only breakpoint value.

http://symfony.com/doc/current/validation/custom_constraint.html

But if this is too hard - we will do it in another version.